conans/client/cache/cache.py

```diff
@@ -58,18 +58,16 @@ def __init__(self, base_folder, store_folder, output):
         super(ClientCache, self).__init__(self._store_folder)
         self.editable_packages = EditablePackages(self.conan_folder)
 
-    def package_layout(self, ref, short_paths=False):
+    def package_layout(self, ref, short_paths=False, *args, **kwargs):
         assert isinstance(ref, ConanFileReference), "It is a {}".format(type(ref))
         edited_ref = self.editable_packages.get(ref.copy_clear_rev())
         if edited_ref:
             base_path = edited_ref["path"]
             layout_file = edited_ref["layout"]
             return PackageEditableLayout(base_path, layout_file, ref)
         else:
-            check_ref_case(ref, self.store)
-            base_folder = os.path.normpath(os.path.join(self.store, ref.dir_repr()))
-            return PackageCacheLayout(base_folder=base_folder,
-                                      ref=ref, short_paths=short_paths)
+            return super(ClientCache, self).package_layout(ref, short_paths=short_paths,
```

Comment:
I'd prefer to keep this here, not in parent class, as it is going to be decoupled client-server (and the client side joined to this CLientCache class)

Reply:
Ok, I will revert it.